src/contour/ContourApp.cpp

```diff
@@ -231,6 +235,18 @@ ContourApp::ContourApp() :
     signal(SIGSEGV, segvHandler);
 #endif
 
+#if defined(_WIN32)
+    // Enable VT output processing on Conhost.
+    HANDLE stdoutHandle = GetStdHandle(STD_OUTPUT_HANDLE);
+    DWORD savedModes{}; // NOTE: Is it required to restore that upon process exit?
+    if (GetConsoleMode(stdoutHandle, &savedModes) != FALSE)
+    {
+        DWORD modes = savedModes;
+        modes |= ENABLE_VIRTUAL_TERMINAL_PROCESSING;
+        SetConsoleMode(stdoutHandle, modes);
```

Comment:
I assumed one has to explicitly enable VT output processing, but for some reason it also worked without on my host. I think it should still stay in here just to be sure it is enabled? 

NOTE: I am not sure i should restore previous modes upon exit, or whether conhost does not care.

Reply:
To make it work reliably, this is needed. However this is some situation that it works without manually setting `ENABLE_VIRTUAL_TERMINAL_PROCESSING`, which maybe considered bug or design? There's some issue in WT repo discussing this but I can't find them now.

You don't need to restore previous mode. Conhost will survive.